modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java

```diff
@@ -149,10 +186,16 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
     // Translate to english
     Boolean translate = getTranslationMode(mediaPackage, workflowInstance);
 
-    for (Track track : tracks) {
-      createSubtitle(track, languageCode, mediaPackage, tagsAndFlavors, appendSubtitleAs, translate);
+    // Subtitles creation
+    if (sourceTagList.isEmpty() && (tracks.length > 1)) {
+      logger.info("Multiple tracks found but no source tags specified");
+      logger.info("Using the first track in the flavor list");
```

Comment:
If it's one message, combine it into one logger statement. Otherwise, you might end up with other log statements in between these two.